listener.go

```diff
@@ -187,7 +187,7 @@ func (l *listener) Close() {
 	if l.consumerGroup != nil {
 		err := l.consumerGroup.Close()
 		if err != nil {
-			ErrorLogger.Printf("Error while closing sarama consumerGroup: %s", err.Error())
+			ErrorLogger.Println(err, "error", "error while closing sarama consumerGroup")
```

Comment:
```suggestion
			ErrorLogger.Println(err, "error", "unable to close sarama consumerGroup")
```

Same for the others change of the same type to avoir reading: "error": "error while..." but it's more an esthetic thing...